app/.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Right, I think that example configuration location is misleading with `app` they probably mean `your-applications-root-path`. I had to move the config directive into `.bundle/config` to install the gems via https.